modules/social_features/social_event/src/Form/EventSettingsForm.php

```diff
@@ -50,31 +102,44 @@ public function buildForm(array $form, FormStateInterface $form_state) {
       ],
     ];
 
-    /** @var \Drupal\group\Entity\GroupTypeInterface $group_type */
-    foreach (GroupType::loadMultiple() as $group_type) {
+    /** @var \Drupal\group\Entity\GroupTypeInterface[] $group_types*/
+    $group_types = $this->entityTypeManager->getStorage('group_type')->loadMultiple();
+    foreach ($group_types as $group_type) {
       // Check if this group type uses events.
       if ($group_type->hasContentPlugin('group_node:event')) {
         // Add to the option array.
-        $form['enroll']['#options'][$group_type->id()] = $group_type->label();
+        $form['event_display']['enroll']['#options'][$group_type->id()] = $group_type->label();
       }
     }
 
-    $form['address_visibility_settings'] = [
+    $form['event_display']['address_visibility_settings'] = [
       '#type' => 'checkboxes',
       '#title' => $this->t('Address visibility settings'),
       '#options' => [
-        'street_code_private' => $this->t('Only show street and postal code to event enrolees'),
+        'street_code_private' => $this->t('Only show street and postal code to event enrollees'),
       ],
-      '#default_value' => $social_event_config->get('address_visibility_settings'),
+      '#default_value' => $social_event_config->get('address_visibility_settings') ?: [],
     ];
 
-    $form['show_user_timezone'] = [
+    $form['event_display']['show_user_timezone'] = [
       '#type' => 'checkbox',
       '#title' => $this->t('Display user’s time zone in events'),
       '#description' => $this->t('If enabled, user’s own time zone will be displayed after the event date and time.'),
       '#default_value' => $social_event_config->get('show_user_timezone'),
     ];
 
+    $form['event_enrolment'] = [
+      '#type' => 'details',
+      '#title' => $this->t('Event enrollment settings'),
+      '#open' => TRUE,
+    ];
+
+    $form['event_enrolment']['allow_to_disable_enroll'] = [
+      '#type' => 'checkbox',
+      '#title' => $this->t('Allow event organizers to disable event enrolments'),
```

Comment:
I think Enrollment is standard in American English, whereas enrolment is standard in British English, so in this case we should stick to the double ll like u used in the other parts as well.